### PR TITLE
Cql4bonnie remove qdm

### PR DESCRIPTION
**Pull request by mayerm94**
Changes were made in bonnie_bundler to remove QDM Loading and Bundle features. This caused errors/invalidated in certain bonnie tests. This PR addresses those errors/invalidations. Test with
projecttacoma/bonnie_bundler#113

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1117
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. NA
- [x] Tests are included and test edge cases NA Tests modified
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Automated regression test(s) pass

**Reviewer 1:** @jbradl11 

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @aelliott
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Comment by mayerm94** [comment]
Changes were made in bonnie_bundler to remove QDM Loading and Bundle features. This caused errors/invalidated in certain bonnie tests. This PR addresses those errors/invalidations. Test with
https://github.com/projecttacoma/bonnie_bundler/pull/113

 https://jira.mitre.org/browse/BONNIE-1117

File: test/functional/admin_users_controller_test.rb [hunk 1]
```diff
@@ -113,26 +113,6 @@ class Admin::UsersControllerTest  < ActionController::TestCase
     assert_equal 3, JSON.parse(response.body).length
   end
 
-  test "bundle download" do
```
Comment (c-monkey) [inline test/functional/admin_users_controller_test.rb:127]: If we decide to completely remove bundles code from the Bonnie project, then removing this test isn't nearly enough, but in that case let's do that removal in https://github.com/projecttacoma/bonnie/pull/857.

**Review by jbradl11 — CHANGES_REQUESTED** [review]
I believe all bundle code should be removed from Bonnie in this PR. If we leave it as is and accept the corresponding Bonnie_bundler PR, then we will have functions (the 'bundle' function in users_controller) that call non existing functions in Bonnie_Bundler, essentially leaving in broken code. It should be all or nothing, and if we are planning on bringing back bundle export, this code should be commented out with a detailed comment above it

**Review by alexanderelliott121 — DISMISSED** [review]

**Review by alexanderelliott121 — CHANGES_REQUESTED** [review]
There are several rake tasks that reference things removed from bonnie bundler.

There is also export_bundle_view.coffee to consider.

**Review by alexanderelliott121 — APPROVED** [review]

**Review by alexanderelliott121 — DISMISSED** [review]

**Review by jbradl11 — DISMISSED** [review]
Looks good.

**Comment by edeyoung** [comment]
tests aren't passing

**Review by edeyoung — DISMISSED** [review]
address test failures

**Review by edeyoung — APPROVED** [review]
approving based on previous approvals

**Pull request merged**